internal/pkg/core/plugin/process/app.go

```diff
@@ -277,6 +275,7 @@ func (a *Application) setState(s state.Status, msg string, payload map[string]in
 }
 
 func (a *Application) cleanUp() {
+	// nolint:errcheck
 	a.monitor.Cleanup(a.desc.Spec(), a.pipelineID)
```

Comment:
linter complained: Error return value of `a.monitor.Cleanup` is not checked (errcheck)